docs/security/03-06-oauth2-server.md

```diff
@@ -6,6 +6,7 @@ type: Details
 By default, every Kyma deployment comes with an OAuth2 authorization server solution from [ORY](https://www.ory.sh/). The `ory` [component](https://github.com/kyma-project/kyma/tree/master/resources/ory) consists of four elements:
 
 - [Hydra](https://github.com/ory/hydra) OAuth2 and OpenID Connect server which issues access, refresh, and ID tokens to registered clients which call services in a Kyma cluster.
+  + By default Hydra is deployed with a database backend for persistent data management. The database used is the [official Bitnami Postgres Package](https://github.com/helm/charts/tree/master/stable/postgresql), however we use [Official Postgres Docker images](https://hub.docker.com/_/postgres?tab=description) instead of those provided by Bitnami. This is because postges provided Alpine based images, which are lighter and have a reduced attack surface.
```

Comment:
```suggestion
  + By default Hydra is deployed with a database backend for persistent data management. The database used is the [official Bitnami Postgres Package](https://github.com/helm/charts/tree/master/stable/postgresql), however we use [Official Postgres Docker images](https://hub.docker.com/_/postgres?tab=description) instead of those provided by Bitnami. This is because Postgres provided Alpine based images, which are lighter and have a reduced attack surface.
```